From 1dd2cf500f5e9448162a1423acdbd37eee4e2241 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Tue, 16 Jul 2024 14:29:21 +0200
Subject: [PATCH 1219/1695] plat-stm32mp2: scmi_server: fix typos in scmi clock
 list

Some SCMI clock names are incorrect, fix this.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 361dc511e7ffcfd4c7680a81e06d1c52b3011a3f plat-stm32mp2: scmi_server: fix typos in scmi clock list]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I325ee4e95f621b391c0e31b60a54e9ff8b9e4475
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/401854
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/plat-stm32mp2/scmi_server_scpfw.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp2/scmi_server_scpfw.c b/core/arch/arm/plat-stm32mp2/scmi_server_scpfw.c
index efd00402c..bd1bca9a5 100644
--- a/core/arch/arm/plat-stm32mp2/scmi_server_scpfw.c
+++ b/core/arch/arm/plat-stm32mp2/scmi_server_scpfw.c
@@ -280,7 +280,7 @@ static struct stm32_scmi_clk stm32_scmi_clock[] = {
 	CLOCK_CELL(CK_SCMI_TSDBG, CK_KER_TSDBG, "ck_ker_tsdbg", true),
 	CLOCK_CELL(CK_SCMI_KER_ETR, CK_KER_ETR, "ck_icn_m_etr", true),
 	CLOCK_CELL(CK_SCMI_BUS_STM, CK_BUS_STM, "ck_icn_p_stm", true),
-	CLOCK_CELL(CK_SCMI_KER_STM, CK_BUS_STM, "ck_icn_s_stm", true),
+	CLOCK_CELL(CK_SCMI_KER_STM, CK_KER_STM, "ck_icn_s_stm", true),
 };
 
 #else /* CFG_STM32MP21 */
@@ -426,7 +426,7 @@ static struct stm32_scmi_clk stm32_scmi_clock[] = {
 	CLOCK_CELL(CK_SCMI_PLL2, PLL2_CK, "ck_pll2", true),
 	CLOCK_CELL(CK_SCMI_PLL3, PLL3_CK, "ck_pll3", false),
 	CLOCK_CELL(CK_SCMI_LPSRAM1, CK_BUS_LPSRAM1, "ck_icn_lpsram1", true),
-	CLOCK_CELL(CK_SCMI_LPSRAM2, CK_BUS_LPSRAM2, "ck_icn_lpsram2 ", true),
+	CLOCK_CELL(CK_SCMI_LPSRAM2, CK_BUS_LPSRAM2, "ck_icn_lpsram2", true),
 	CLOCK_CELL(CK_SCMI_LPSRAM3, CK_BUS_LPSRAM3, "ck_icn_lpsram3", true),
 	CLOCK_CELL(CK_SCMI_VDERAM, CK_BUS_VDERAM, "ck_icn_s_vderam", true),
 	CLOCK_CELL(CK_SCMI_SYSRAM, CK_BUS_SYSRAM, "ck_icn_s_sysram", true),
@@ -438,7 +438,7 @@ static struct stm32_scmi_clk stm32_scmi_clock[] = {
 	CLOCK_CELL(CK_SCMI_TSDBG, CK_KER_TSDBG, "ck_ker_tsdbg", true),
 	CLOCK_CELL(CK_SCMI_KER_ETR, CK_KER_ETR, "ck_icn_m_etr", true),
 	CLOCK_CELL(CK_SCMI_BUS_STM, CK_BUS_STM, "ck_icn_p_stm", true),
-	CLOCK_CELL(CK_SCMI_KER_STM, CK_BUS_STM, "ck_icn_s_stm", true),
+	CLOCK_CELL(CK_SCMI_KER_STM, CK_KER_STM, "ck_icn_s_stm", true),
 };
 #endif
 
-- 
2.47.2

